_includes/masthead.html

```diff
@@ -144,19 +144,19 @@
       <div class="mobile-menu-wrapper">
         <img id="closeMenuIcon" class="closeMenuIcon" src="{{ '/images/closeMenuIcon.svg' | absolute_url }}" />
         <ul>
-          <li><a href="{{ '/docs' | absolute_url }}">Open Source docs</a></li>
+          <li><a href="{{ '/docs' | absolute_url }}">Documentation</a></li>
           <li>
             <a href="https://wiremock.io?utm_source=wiremock.org&utm_medium=masthead&utm_campaign=homepage_2022_baseline" title="WireMock Cloud">
               <img src="{{ '/images/wiremock-cloud/wiremock_cloud_logo.png' | absolute_url }}" alt="WireMock Cloud" style="vertical-align: middle;height: 28px"/><br>
-              WireMock Cloud docs
+              WireMock Cloud
             </a>
           </li>
-          <li><a href="{{ '/studio/docs' | absolute_url }}">WireMock Studio docs</a></li>
           
-          <li><a href="{{ '/support' | absolute_url }}">Support</a></li>
+          <li><a href="{{ '/support' | absolute_url }}">Need Help?</a></li>
+          <li><a href="https://slack.wiremock.org/" target="_blank">Community Slack</a></li>
           <li><a href="https://github.com/wiremock/wiremock" target="_blank">GitHub</a></li>
-          <li><a href="https://github.com/wiremock/wiremock/blob/master/CONTRIBUTING.md" target="_blank">Contributing</a></li>
-          <li><a href="{{ '/external-resources' | absolute_url }}">Resources</a></li>
```

Comment:
Are we ditching this page?

Reply:
Nope, it is still on the main bar and on the navigation. It just has little sense to keep it on the hamburger menu for mobile devices, because it renders really bad